modules/engines/engine-hdi/src/main/java/com/sap/xsk/hdb/ds/processors/hdi/XSKHDIContainerCreateProcessor.java

```diff
@@ -80,6 +110,10 @@ public void execute(Connection connection, XSKDataStructureHDIModel hdiModel) {
       // Deploy the Content
       this.deployContainerContentProcessor.execute(connection, hdiModel.getContainer(), hdiModel.getDeploy(), hdiModel.getUndeploy());
 
+      // Grant Privileges to the default role
+      this.grantPrivilegesDefaultRoleProcessor.execute(connection, hdiModel.getContainer(), Configuration.get("HANA_USERNAME"), hdiModel.getDeploy(),
```

Comment:
you can extract Configuration.get("HANA_USERNAME") in local variable and reuse it, since it is also used on line 88